third_party/ScubaDiver/src/DllEntry.cs

```diff
@@ -38,10 +57,15 @@ public static int EntryPoint(string pwzArgument)
     // then we need to allocate a console and redirect STDOUT to it.
     Logger.RedirectConsole();
 
-    //
+    // Unload the native bootstrapper DLL to free up the file handle.
+    if (!UnloadBootstrapper())
+    {
+      Logger.Debug("[EntryPoint] Failed to unload Bootstrapper.");
+      return 1;
+    }
+
```

Comment:
Client registration is currently broken; will need to investigate whether the bootstrapper assembly is being unloaded.